### DISCUSSION TITLE
How to copy files to ish file system from PC?

### DISCUSSION THREAD

**Discussion by kernel-cyrus** [ish-app/ish#1195] [Q&A] [answered]
I have a windows pc with itunes installed, the iSH APP is shown in itunes. After I copy files to the iSH APP, I can't find them on my phone. Does anyone knows a better way to easily copy files to ish file system from PC? That would be very helpful, thank you!

**Comment by jopilote** [comment, marked as answer]
first enable* the iSH directory in "Files" ,  if you haven't  you cant access any  the / of alpine+iSH.

I don't use  itunes tho.   Files in "files app" can be accesed from iSH with read+write permissions.

* in "files" + Browse:  click upper right setings "(...)" , click "edit' , enable (green) iSH
